src/app/shared/recognize-video-type-id.ts

```diff
@@ -0,0 +1,226 @@
+
+const getSrc = require('get-src');
```

Comment:
nie komentuję tego pliku bo to będziesz przepisywał, ale w sumie jestem zakoczony że kompilator tsa nie płacze o to require

Reply:
jak działa to nie ruszaj ;) przynajmniejh na razie